src/sql/azdata.d.ts

```diff
@@ -74,13 +74,6 @@ declare module 'azdata' {
 		export function getProvider(namespaceId: string): Thenable<CredentialProvider>;
 	}
 
-	/**
-	 * Namespace for serialization management global methods
-	 */
-	export namespace serialization {
```

Comment:
I verified no published extension on github / internally uses this and the other APIs here. It was always a placeholder that wasn't implemented up to now 😢